scripts/shell.nix

```diff
@@ -8,6 +8,7 @@ mkShell {
     # NOTE using the most recent nix for sticky nix flake lock update feature
     nix
     python3
+    moreutils
```

Comment:
Do we actually use the tools from `moreutils` in this PR? 🤔

Reply:
yes, `proto` script requires `sponge` utility from `moreutils`

Reply:
Got it 👌